extensions/graphviz_support/graphviz_support/method/method.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from typing import List
+
+
+class Method:
+    """
+    Represents a node in the graph.
```

Comment:
Do you want to add in the description of the attributes in the class.

Reply:
Done